prompt2model_demo.py

```diff
@@ -351,46 +353,113 @@ def main():
+            )
             try:
-                train_batch_size = int(line)
-                assert 0 < train_batch_size
+                assert line in ["y", "n"]
                 break
             except Exception:
-                line_print("The training batch size must be greater than 0.")
+                line_print("The answer should be either y or n")
         time.sleep(1)
 
-        while True:
-            line = input("Enter the number of epochs to train for:")
-            try:
-                num_epochs = int(line)
-                break
-            except ValueError:
-                line_print("Invalid input. Please enter a number.")
-        time.sleep(1)
+        if line == "y":
+            line_print("Moving with training and hyperparameter selection.")
+            max_num_epochs = input(
+                "Enter max number of epochs: Press enter to use default value: "
+            )
+            num_trials = input(
+                "Enter the number of trials to conduct hypeparamter search. Press enter to use default value: "  # noqa E501
+            )
+            max_batch_size = input(
+                "Enter the max batch size Press enter to use default: "
+            )
+
+            max_num_epochs = (
+                DEFAULT_HYPERPARAMETERS_SPACE["max_num_train_epochs"]
+                if max_num_epochs == ""
+                else eval(max_num_epochs)
+            )
+            num_trials = 1 if num_trials == "" else eval(num_trials)
+            max_batch_size = (
+                DEFAULT_HYPERPARAMETERS_SPACE["per_device_train_batch_size"][0]
+                if max_batch_size == ""
+                else eval(max_batch_size)
+            )
+
+            trainer = GenerationModelTrainer(
+                status["model_name"],
+                has_encoder=True,
+                executor_batch_size=max_batch_size,
+                tokenizer_max_length=1024,
+                sequence_max_length=1280,
+            )
+            args_output_root = Path("result/training_output")
+            args_output_root.mkdir(parents=True, exist_ok=True)
+            line_print("Starting training.")
+
+            trained_model, trained_tokenizer = OptunaParamSelector(
+                n_trial=num_trials,
+                trainer=trainer,
+            ).select_from_hyperparameters(
+                training_datasets=training_datasets,
+                validation=validation_datasets,
+                hyperparameters={
+                    "max_num_train_epochs": max_num_epochs,
+                    "per_device_train_batch_size": [max_batch_size],
+                },
+            )
+            # FIXME: The batch size selection needs some more smarter way to
+            # avoid cuda out of error and automatic list
+            # creation based on the max size with some intervals
+            train_batch_size = max_batch_size  # need to change this
+
+        else:
+            line_print("Going with normal training")
```

Comment:
Suggestion for cleaning up the language here:

```suggestion
            line_print("Starting training without hyperparameter selection.")
```

Reply:
@Anindyadeep could you revise this text or accept my suggestion? I think the current language sounds overly casual